lib/graphql/schema.ts

```diff
@@ -500,6 +498,7 @@ export type Customer = {
   __typename?: 'Customer';
   id: Scalars['ID'];
   name?: Maybe<Scalars['String']>;
+  email?: Maybe<Scalars['String']>;
```

Comment:
We have this email in the Customer model and in the customers table but on the frontend side there is no input to fill this information in 😅 . So maybe we should clean it up on the BE or add a new input field on the FE. Checking with Dani and Justin.